vaadin-login-flow-parent/vaadin-login-flow/src/main/java/com/vaadin/flow/component/login/LoginOverlay.java

```diff
@@ -231,6 +237,130 @@ public String getDescription() {
         return getElement().getProperty("description");
     }
 
+    /**
```

Comment:
DX Question: Would it make sense to have `addCustomField(InputField)` to improve discoverability of the new API? (using InputField instead of Component here to enforce "proper" fields)

Reply:
Yes, initially I was thinking about this. Thanks for the idea about using `InputField`, it sounds good to me.